### ISSUE TITLE
Where I put ##From and ##cc comands?

### ISSUE THREAD

**Issue by empereira** [tomolimo/mailbehaviours#3]
hello,

Where I put ##From and ##cc commands? On the mail body after a forward press button?

Ex:

---------- Forwarded message ---------
##From: Test <test@test.org>
Date: qua., 6 de jan. de 2021 às 19:00

**Comment by tomolimo** [comment]
Hello @empereira 

Yes, see this example: https://github.com/tomolimo/mailbehaviours/blob/main/MailBehaviours%20sample.png

For the ##From: only the first ##From will be taken into account.
For the ##CC: all the lines will be used to add watcher users (or groups)

Thank you
Regards,
Tomolimo

**Comment by empereira** [comment]
Thanks for the answer @tomolimo!!!!

Does it check all the "##From" or only the "##"? I ask because I'm Brazilian and the "From" to us is "De".

**Comment by tomolimo** [comment]
Hello @empereira,
It checks the "##from" using the following regex:

`'/##From\s*:\s*(["\']?(?\'last\'[\w.\-\\\' ]+)[, ]\s*(?\'first\'[\w+.\-\\\' ]+))?.*?(?\'email\'[\w_.+\-]+@[\w\-]+\.[\w\-.]+)?\W*$/imu'`

Thank you,
Regards,

Tomolimo

**Comment by empereira** [comment]
Ok. If I change the `@param string` to `@param string $search "from" or "de" or "CC"`, it's work?

Btw, great work with the plugin!!! Thanks!!!

**Comment by tomolimo** [comment]
yes it should but it means that next release of the plugin will cancel your modification.

**Comment by empereira** [comment]
No problem!! :+1: 

Or maybe you could add an input or something so that we can put the "From", in my case the "De" according to the country of origin of each user? :D

**Comment by rubensaraiva10** [comment]
Hello, I got some problems, I use Glpi with Ldpa, and my email enters but not for the correct user. I am using ##From: Test Test@test.com but is not working.

**Comment by tomolimo** [comment]
Hello @rubensaraiva10 

Please do not use existing issues to post new incidents/questions... Otherwise at the end issues are mixed up, and it is impossible to help you correctly.
Could you open a new issue?

Thank you
Regards,
Tomolimo

**Comment by juanzu** [comment]
> Hello @empereira
> 
> Yes, see this example: https://github.com/tomolimo/mailbehaviours/blob/main/MailBehaviours%20sample.png
> 
> For the ##From: only the first ##From will be taken into account.
> For the ##CC: all the lines will be used to add watcher users (or groups)
> 
> Thank you
> Regards,
> Tomolimo

Hello how are you I am from Colombia and I am using a translator. I am trying to configure glpi to create tickets by mail. Investigating I found your plugin but I don't know how to use it. I have tried to put ## from and ## cc but no cases are created. Could you help me please

**Comment by tomolimo** [comment]
> 
> Hello how are you I am from Colombia and I am using a translator. I am trying to configure glpi to create tickets by mail. Investigating I found your plugin but I don't know how to use it. I have tried to put ## from and ## cc but no cases are created. Could you help me please

Hello @juanzu 
Could you post in a new issue rather than mixing several topics?
Thank you
Regards,
Tomolimo
